mysql-test/suite/auth_sec/t/install_keyring_vault.test

```diff
@@ -0,0 +1,20 @@
+--disable_ps_protocol
```

Comment:
Only common tests are located in udf tests, rest of the test have a lot of common functionality, but there are no common test cases in separate files for keyring_file that I could include in keyring_vault tests. I would need to change keyring_file tests - so the would be build of sort of includes and those includes add to keyring_vault tests. However, this means changing keyring_file mtr tests which would mean that merges would become quite difficult.